lib/helpers/utils.js

```diff
@@ -6,6 +6,19 @@ const REQUEST_TIMEOUT = process.env.REQUEST_TIMEOUT || 10000; // 10s
 const REQUEST_MAX_RETRY = process.env.REQUEST_MAX_RETRY || 3;
 const REQUEST_RETRY_DELAY = process.env.REQUEST_RETRY_DELAY || 5000; // 5s
 
+module.exports.columnToLetter = function columnToLetter(column) {
```

Comment:
why you don want to use https://github.com/elasticio/gspreadsheets/blob/read-sheet-extend/lib/common.js#L1 ??

Reply:
I'm using it
I've just added module.exports to it